rest/admin_api_test.go

```diff
@@ -33,6 +33,121 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+// Test warnings being issued when a new channel is created with over 250 characters - CBG-1475
+func TestChannelNameSizeWarning(t *testing.T) {
```

Comment:
This test is fairly hard to read. There is a lot going on. 
Couple comments below on this.